apps/arena/lib/arena/configuration.ex

```diff
@@ -184,6 +185,15 @@ defmodule Arena.Configuration do
     }
   end
 
+  defp parse_crate(crate) do
```

Comment:
It seems parse_pool, parse_bush, and parse crate do the same thing, we can have just one function for them 🤔 . What do you think?

Reply:
Sounds good! 🪨